test/e2e/capic_test.go

```diff
@@ -121,3 +121,17 @@ var _ = Describe("Should be able to create a cluster with 1 control-plane using
 		}
 	})
 })
+
+var _ = Describe("Should be able to create a cluster with 1 control-plane and 2 workers using an image selector", func() {
```

Comment:
Is it okay with long e2e runs?

We already have cases; if you want, you can set the flavor to one of them.

Another suggestion would be to label test cases and control them with  PR labels.
for example capmox, have something like this:
https://github.com/ionos-cloud/cluster-api-provider-proxmox/pull/352#issuecomment-2520339157

But.  the whole e2e topic is not related to this PR.

Reply:
i think both code paths (image id and image selector) should be covered. i changed the conformance tests to use image selectors, so i think this test case can be removed.